.editorconfig

```diff
@@ -0,0 +1,14 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = false
```

Comment:
> At least in this project, there were no trailing newlines in the files.

Ok. Thats probably my fault. But still, if we are adding rules to the repo I want the rules to be correct.